simtools/applications/simulate_prod.py

```diff
@@ -140,25 +139,22 @@ def _parse(description=None):
     )
     config.parser.add_argument(
         "--from_azimuth_direction",
-        help="Direction from which the primary reaches the atmosphere",
-        type=str.lower,
+        help=(
+            "Direction from which the primary reaches the atmosphere."
+            "It can be in degrees between 0 and 360 or one of (north, south, east, west)."
```

Comment:
Is this the right place to mention that North=0 deg (or 180deg) and that it is clockwise / anticlockwise?

Reply:
Yes, I will add that as well.